plugins/freecycle/functions.php

```diff
@@ -3069,4 +3074,54 @@ function escape_html_special_chars($text, $charset = 'utf-8'){
 	$ngwords = array("<", ">", ";");
 	$nongtext = str_replace($ngwords, "", $text);
 	return htmlspecialchars($nongtext, ENT_QUOTES, $charset);
+}
+
+
+//Reservation処理の実装
+//new reservation
+function reservationAdd(){
+	global $wpdb;
+	global $table_prefix;
+	global $user_ID;
+	$item_id=$_GET["item_id"];
```

Comment:
Please make your new functions available by not ajax, too. For example,

function reservationAdd($item_id, $bookfair_id, $status){
...
}
  function reservationAddAjax(){
      $item_id =  isset($_GET["item_id])?$_GET["item_id]:"";
...
     reservationAdd($item_id, $bookfair_id, $status);
}